File: src/shared/XXXXX.Domain/Models/PermissionAdmins/PermissionAdminCategory.cs
using System;
using System.Collections.Generic;

namespace XXXXX.Domain.Models
{
    public class PermissionAdminCategory: ITranslatable<TranslationPermissionAdminCategory>
    {
        public string Label { get; set; }
        public string Prefix { get; set; }
        public List<TranslationPermissionAdminCategory> Translations { get; set; }
    }

    public class TranslationPermissionAdminCategory: ITranslation
    {
        public string LanguageCode { get; set; }
        public string Label { get; set; }
    }
}

File: src/app/admin/XXXXX.Admin.UI/src/domain/models/permissionAdmins/permissionAdminCategory.ts
export class PermissionAdminCategory {
    label: string;
    prefix: string;

    constructor(params: PermissionAdminCategoryDTO) {
        this.label = params.label;
        this.prefix = params.prefix;
    }
}

export interface PermissionAdminCategoryDTO {
    label: string;
    prefix: string;
}